tools/genomespace/genomespace_exporter.xml

```diff
@@ -1,54 +1,42 @@
 <?xml version="1.0"?>
-<tool name="GenomeSpace Exporter" id="genomespace_exporter" require_login="True" version="0.0.4">
+<tool name="GenomeSpace Exporter" id="genomespace_exporter" version="0.0.4">
     <description> - send data to GenomeSpace</description>
-    <command interpreter="python">genomespace_exporter.py 
-        --genomespace_site "prod"
-        #assert $__user__, Exception( 'You must be logged in to use this tool.' )
-        #set $username = $__user__.preferences.get( 'genomespace_username', None )
-        #set $token = $__user__.preferences.get( 'genomespace_token', None )
-        #assert None not in ( $username, $token ), Exception( 'You must associate a GenomeSpace OpenID with your account and log in with it.' )
+    <command interpreter="python">genomespace_exporter.py
+        #set $target_folder = $genomespace_browser.split("^")[0]
+        #set $token = $genomespace_browser.split("^")[1] or $__user__.preferences.get( 'genomespace_token', None )
```

Comment:
@mvdbeek Thanks, I'm attending to these now. Is there a way to make non data-source tools pass in a json_parameter_file with all params included (as in the importer here: https://github.com/galaxyproject/galaxy/pull/1814/files#diff-1090cafa7e8b7558568423c4b7ff4dd6R12)? I found that only data-source tools do this. For non-data source tools, I had to override the exec_before_job hook to manually generate a json param dict: https://github.com/galaxyproject/galaxy/pull/1814/files#diff-a9715b311879402b8b86a4013e4ec56bR67

If there's a way to do this, it would significantly simplify the code, and allow for a secure way to pass in the token. If not, I'll fall back to #open('token.txt', 'w').write($token)